web/about.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<title>About YOU.</title>
+
+<meta name="viewport" content="width=device-width">
+<meta name=”robots” content="index, follow">
+<meta name="author" content="Anonymous">
+<meta itemprop="name" content="About You.">
+<meta itemprop="description" content="About You. -- The privacy-first period-tracking app">
+<meta itemprop="image" content="https://youperiod.app/images/logo-social-small.png">
+<meta name="twitter:card" content="summary">
+<meta name="twitter:title" content="About You.">
+<meta name="twitter:description" content="About You. -- The privacy-first period-tracking app">
+<meta name="twitter:image" content="https://youperiod.app/images/logo-social-small.png">
+<meta name="twitter:image:alt" content="You. App">
+<meta name="title" property="og:title" content="About You.">
+<meta name="description" property="og:description" content="About You. -- The privacy-first period-tracking app">
+<meta name="image" property="og:image" content="https://youperiod.app/images/logo-social-large.png">
+<meta name="url" property="og:url" content="https://youperiod.app/">
+<meta name="site_name" property="og:site_name" content="You.">
+<meta name="type" property="og:type" content="website">
+
+<link rel="icon" type="image/x-icon" href="/favicon.ico" />
```

Comment:
This line needs a "canonical" meta tag like index has, and it should, of course, point to "https://youperiod.app/about.html".